test/checkpoint/test_checkpoint.c

```diff
@@ -278,21 +281,33 @@ wt_connect(const char *config_open)
           g.checkpoint_slow_timing_stress ? "checkpoint_slow" : ""));
     }
 
+    /*
+     * Randomly decide on the eviction rate (fast or default).
+     */
+
+    if ((__wt_random(&rnd) % 15) % 2 == 0) {
```

Comment:
Personally I would prefer the method of adding an extra configuration of `-e` to set the eviction fast or default. Let me know what you think around this. Also @kommiharibabu what are your opinions?

Reply:
Correct me if I'm wrong @kommiharibabu but I think the last time we had a conversation around this, we discussed that having it randomly was workable to achieve the desired result and takes less time to implement.

Reply:
Yes, I prefer to go with the random approach as this way the high eviction rate is tested with all the existing options. That doesn't mean I don't like the external option approach, but I don't think it is required.